### PR TITLE
Revamping project for worldcup

### PR DESCRIPTION
**Pull request by Siggnja**
- Initilize the project for world cup 2022
- Adds flags for missing teams
- Creates seed data with all matches and teams
- Adds a cronjob to fetch scores every hour

**Todo**
- Get postgres db up and running
- Deploy to gisk.app

### CODE REVIEW

File: .github/workflows/update-scores-cron.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+         run: yarn --frozen-lockfile
+
+      - name: Update scores
+        run: DATABASE_URL=${{env.DATABASE_URL}} npx ts-node ./db/updateScores.ts
+        env:
+          DATABASE_URL: ${{ secrets.DATABASE_URL }}
```
Comment (Siggnja) [inline .github/workflows/update-scores-cron.yml:30]: We need to add the postgres url to github action secrets.
@gunnartorfis

File: db/updateScores.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      return;
+    }
+
+
+    await db.match.update({
+      where: {
+        homeTeamId_awayTeamId_kickOff: {
+          homeTeamId: hometeam?.id as string,
+          awayTeamId: awayTeam?.id as string,
+          kickOff: newMatch.datetime
```
Comment (gunnartorfis) [inline db/updateScores.ts:27]: I'm wondering if we should store the match ID we get from the API and use that to update?

**Comment by gunnartorfis** [comment]
Awesome work as always 🚀

**Review by gunnartorfis — APPROVED** [review]

File: db/updateScores.ts (hunk 2 above)
Comment (Siggnja) [reply db/updateScores.ts:27]: Yeah, I thought about that as-well. We can try it out.

**Pull request merged**